inst/tests/tests.Rraw

```diff
@@ -10771,9 +10773,9 @@ test(1743.241, fread("a,b,c\n2,2,f", colClasses = list(character="c", integer="b
 test(1743.242, fread("a,b,c\n2,2,f", colClasses = c("integer", "integer", "factor"), drop="a"), data.table(b=2L, c=factor("f")))
 
 ## POSIXct
-test(1743.25, fread("a,b,c\n2015-06-01 11:00:00,1,ae", colClasses=c("POSIXct","integer","character")), data.table(a=as.POSIXct("2015-06-01 11:00:00"),b=1L,c="ae"))
+test(1743.25, fread("a,b,c\n2015-06-01 11:00:00,1,ae", colClasses=c("POSIXct","integer","character")), data.table(a=as.POSIXct("2015-06-01 11:00:00", tz='UTC'),b=1L,c="ae"))
```

Comment:
three more tests broken because I forced `tz='UTC'` in the implementation.

this is making me reconsider that -- should we default to not setting `'tzone'`?

Reply:
isn't this going to be a breaking change? maybe we could relax that, and keep old behavior for coming release and change it in next release?

Reply:
yes i agree. not even 100% sure we should change the behavior -- if so we need to be very explicit that it's a break with `base`